src/components/related-page/related-page.js

```diff
@@ -7,6 +7,45 @@ import BookImage from '../book-image/book-image';
 import GlossaryImage from '../glossary-image/glossary-image';
 import ExampleImage from '../example-image/example-image';
 
+class PlaygroundImage extends React.Component {
```

Comment:
Should we put `PlaygroundImage` into its own component and import to be consistent with `GlossaryImage`, `ExampleImage`, etc. above?

Reply:
@colleenmcginnis my initial thought was that we likely wouldn't use the PlaygroundImage elsewhere, but that could change. I'll break it out into its own component!